app/registry.hs

```diff
@@ -164,21 +185,31 @@ newtype PublicKeyFile =
   deriving (Eq, Ord, Show, IsString)
 
 data Submission = Submission
-  { sId :: !Id
+  { sOwner :: !Owner
+  , sName :: !Name
+  , sDescription :: !(Maybe Description)
```

Comment:
Ok, so in this commit (besides the less interesting parts) we:

1. remove the stake pool identifier (presumably to add it later)
2. add the owner identifier
3. change the filename relationship to be "filename = owner Id"

cc @KtorZ

Reply:
>  (presumably to add it later)
No. The trick is that a pool can be owned be several owners. So, who's going to sign it? Using the VRF to produce a signature could be tricky and wasn't recommended by our cryptographers in this context. 

Imagine the following scenario: 

> I am a "malicious actor" and I submit some metadata for stake pool A, which I don't own. I do however provide a valid public key and a valid signature that can be verified; yet, my public key is not part of the owners of pool A. 

Since the registry doesn't do any on-chain validation, it can't verify whether a particular submission is valid with this regards.

So, by making the metadata about `owner` instead and having the consumers (e.g. wallet backend) do the mapping (owner -> stake pool) itself because they have access to the on-chain data, we prevent this kind of hi-jacking.  


Having said that, yes, the changes are as you described.